tests/sentry/uptime/subscriptions/test_subscriptions.py

```diff
@@ -773,16 +784,18 @@ def test_disable_failed(self, mock_resolve_uptime_issue, mock_disable_seat) -> N
             self.tasks(),
             self.feature(UptimeDomainCheckFailure.build_ingest_feature_name()),
         ):
-            proj_sub = create_project_uptime_subscription(
+            detector = create_uptime_detector(
                 self.project,
                 self.environment,
                 url="https://sentry.io",
                 interval_seconds=3600,
                 timeout_ms=1000,
                 mode=UptimeMonitorMode.MANUAL,
-                uptime_status=UptimeStatus.FAILED,
```

Comment:
Should we keep passing this for now?

Reply:
Actually I think we can get rid of this parameter completely, nothing else is passing it into this function